x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/action/EsqlCapabilities.java

```diff
@@ -109,7 +86,6 @@ private static Set<String> capabilities() {
         for (NodeFeature feature : new EsqlFeatures().getHistoricalFeatures().keySet()) {
             caps.add(cap(feature));
         }
-        caps.add(STRING_LITERAL_AUTO_CASTING_TO_DATETIME_ADD_SUB);
```

Comment:
I always wondered why this was registered after the others, as if it mattered somehow?

Reply:
I have no idea, but my feeling is that the order doesn't really matter